.github/workflows/windows_build.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     runs-on: 'windows-latest'
     strategy:
       matrix:
-        python-version: ['3.12']
+        python-version: ['3.13']
```

Comment:
More tools are supporting `.python-version` file or in `pyproject.toml`, so you only have one place to change.

uv: https://docs.astral.sh/uv/concepts/python-versions/#python-version-files
actions/setup-python: https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-file-input